src/components/common/NavBar.js

```diff
@@ -5,27 +5,31 @@ import {
   InstagramOutlined,
   YoutubeOutlined,
   LinkedinOutlined,
+  ThunderboltOutlined,
 } from '@ant-design/icons';
+import { Link } from 'react-router-dom';
 
 export default function MainNav() {
   return (
     <nav className="mainNav">
       <div className="nav navLogo">
-        <h1 className="navTitle">CODERHEROES</h1>
+        <Link to="/">
+          <h1 className="navTitle">{<ThunderboltOutlined />}CODERHEROES</h1>
```

Comment:
On our Figma mockup for the nav bar, we used the filled thunderbolt icon(which would be `<ThunderboltFilled />`). I think it would look better if we used that version for the logo.

Reply:
Thank you. I will implement this and push this change before Monday.